### ISSUE TITLE
Is this actively maintained?

### ISSUE THREAD

**Issue by btford** [jimrhoskins/ngmodules.org#32]
I haven't seen much movement on this repo in a year. Is this actively maintained? Do you want help with this project?

**Comment by jimrhoskins** [comment]
Sadly I have had not had the time to maintain it as of late. I've put it out there that I'm looking for another maintainer, without much interest. Definitely deserves more attention than I can give it right now. If the community wants to take it over, or people want to actively contribute, please let me know. Email me at jim@jimhoskins.com

**Comment by locnguyen** [comment]
I run the [AngularJS-OC meetup](http://www.meetup.com/angularjs-oc) in SoCal. I've been trying to find projects to collaborate on with my members for hack nights. This might be a great fit if I can Agularize ngmodules and help newbies learn the framework.

@btford Are there features you'd like to see implemented?

**Comment by btford** [comment]
@locnguyen yes. I'll file issues for these separately, but basically I want to mirror the work I'm doing on https://github.com/btford/pierogi, https://github.com/btford/readme-good, etc. I want automated metrics for module quality.

See:
- #33 
- #34
